### PR TITLE
Update cookbook for Chef 13

### PR DESCRIPTION
**Pull request by ledgr**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -1,25 +1,21 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-Vagrant.configure("2") do |config|
+Vagrant.configure('2') do |config|
```
Comment (ton31337) [inline Vagrantfile:4]: just 2?

File: metadata.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-license          "All rights reserved"
-description      "Installs/Configures php-ioncube"
+name             'php-ioncube'
+maintainer       'Andriy Samilyak'
+maintainer_email 'samilyak@gmail.com'
+license          'All rights reserved'
+description      'Installs/Configures php-ioncube'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "0.2.0"
+version          '0.2.2'
```
Comment (ton31337) [inline metadata.rb:7]: why not 0.2.1?

Comment (ledgr) [reply metadata.rb:7]: Because there already is '0.2.1' version: https://supermarket.chef.io/cookbooks/php-ioncube

File: Vagrantfile (hunk 1 above)
Comment (ledgr) [reply Vagrantfile:4]: Yes

Comment (ton31337) [reply Vagrantfile:4]: I mean why not integer?

File: recipes/install.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 include_recipe 'php-ioncube::default'
 
-php_ioncube_install "ioncube" do
-    action :install
+php_ioncube_install 'ioncube' do
```
Comment (ton31337) [inline recipes/install.rb]: `php_ioncube_install 'ioncube'` not enough?

File: Vagrantfile (hunk 1 above)
Comment (ledgr) [reply Vagrantfile:4]: Documentation uses string, vagrant file generator generates it as string too.

**Review by ton31337 — APPROVED** [review]
it's not going to be pushed to upstream, thus nevermind

**Pull request merged**